commands/backups/restore.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+
+function dropboxURL (url) {
+  // Force a dump file to download instead of rendering as HTML
+  // file by specifying the dl=1 param for Dropbox URL
+  if (url.match(/www.dropbox.com/) && !url.endsWith('dl=1')) {
+    if (url.endsWith('dl=0')) url = url.replace('dl=0', 'dl=1')
+    else if (url.includes('?')) url += '&dl=1'
```

Comment:
Note that an unencoded `?` can appear in the password or username field (e.g., `https://?:?@dropbox.com/foo`). Again, probably not a real-world problem.